# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/platforms/agnostic/AgnosticDecoderModule.h
# Commit: f9337268523c
# Full Hash: f9337268523c6607228fda7fa659fd5a3d939dad
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2020-10-21 09:51:28
# Regressor Bug: 1595994
# File Overlap Count: 1
# Description:
#   Bug 1595994 - P1E. Don't use libvpx in the RDD process if video contains an alpha plane. r=mattwoodrow
#   
#   Until bug 1668840 is fixed, we can't allocate a SharedRGBImage without a PImageBridge
#   
#   Depends on D92233
# ==============================================================================

diff -r 7a990fa02246 -r f9337268523c dom/media/platforms/agnostic/AgnosticDecoderModule.h
--- a/dom/media/platforms/agnostic/AgnosticDecoderModule.h	Tue Oct 20 23:30:04 2020 +0000
+++ b/dom/media/platforms/agnostic/AgnosticDecoderModule.h	Tue Oct 20 23:30:18 2020 +0000
@@ -17,6 +17,8 @@
 
   bool SupportsMimeType(const nsACString& aMimeType,
                         DecoderDoctorDiagnostics* aDiagnostics) const override;
+  bool Supports(const SupportDecoderParams& aParams,
+                DecoderDoctorDiagnostics* aDiagnostics) const override;
 
  protected:
   AgnosticDecoderModule() = default;